neuromation/cli/config.py

```diff
@@ -68,6 +68,20 @@ def config() -> None:
     click.echo(fmt(cluster_quota))
 
 
+@command()
+@async_cmd()
+async def add_quota(root: Root) -> None:
+    """
+    Print instructions for increasing quota for current user
+    """
+    user_name = root.client.config.username
+    cluster_name = root.client.config.cluster_name
+    click.echo(
+        f"In order to increase your quota, please navigate to "
+        f"https://neuro.payments.com/{user_name}/{cluster_name}?pay=usd100"
```

Comment:
Should this URL be a part of a server provided config?

Reply:
Eventually, yes. But this particular part is not finished yet (and the command is disabled). It will be completely released after Evil Martians complete their part and it's tracked via a separate issue.